### PR TITLE
fix(suite): Refactor EVM banners

### PR DESCRIPTION
**Pull request by jvaclavik**
## Related Issue

Resolve 

## Screenshots:

Screenshot 2024-09-17 at 15 19 00
Screenshot 2024-09-17 at 15 18 41

### CODE REVIEW

File: packages/suite/src/components/wallet/WalletLayout/AccountBanners/BannerPoints.tsx [hunk 1]
```diff
@@ -11,24 +10,20 @@ const Point = styled.div`
...
     if (points.length === 0) return null;
 
     return (
-        <>
+        <div>
             {points.map((point, i) => (
-                <Point key={i}>
-                    <Icon name="check" size={20} />
+                <Point key={`bullet-${i}-${point}`}>
+                    {points.length === 1 ? '' : '• '}
```
Comment (seibei-iguchi) [inline packages/suite/src/components/wallet/WalletLayout/AccountBanners/BannerPoints.tsx]: ul and li? Make the bullet via css?

Comment (jvaclavik) [reply packages/suite/src/components/wallet/WalletLayout/AccountBanners/BannerPoints.tsx]: Good point 👍 done

File: packages/suite/src/components/wallet/WalletLayout/AccountBanners/BannerPoints.tsx [hunk 2]
```diff
@@ -1,34 +1,28 @@
...
-import { spacingsPx, typography } from '@trezor/theme';
+import { typography } from '@trezor/theme';
 
 interface BannerPointsProps {
     points: ReactNode[];
 }
 
-const Point = styled.div`
-    display: flex;
+const ListItem = styled.li<{ $hasBullet: boolean }>`
```
Comment (seibei-iguchi) [inline packages/suite/src/components/wallet/WalletLayout/AccountBanners/BannerPoints.tsx]: No need for the prop. Use `&:only-child`

Comment (jvaclavik) [reply packages/suite/src/components/wallet/WalletLayout/AccountBanners/BannerPoints.tsx]: Done 👍

**Review by seibei-iguchi — APPROVED** [review]

**Pull request merged**